### PR TITLE
Adding unknown unicast, broadcast and unknown multicast flood groups to VLAN

### PR DESCRIPTION
**Pull request by karthik-krishnamurthy**
Adding unknown unicast, broadcast and unknown multicast flood groups to VLAN as discussed in the OCP weekly meeting

### CODE REVIEW

**Comment by karthik-krishnamurthy** [comment]
@lguohan please review

**Comment by JipanYanga** [comment]
Now there are default L2MC groups for unknown unicast, unknown multicast and broadcast per VLAN,  how to get the L2MC group ID for each of them? thanks.

**Comment by karthik-krishnamurthy** [comment]
Modified the comments in the pull request  and explained the behavior in the comments.
@lguohan Please let me know if these are fine.

File: inc/saivlan.h [hunk 1]
```diff
@@ -266,14 +266,63 @@ typedef enum _sai_vlan_attr_t
...
+     * packets to all vlan members would be restored.
+     *
+     * @type sai_object_id_t
      * @flags CREATE_AND_SET
-     * @default false
+     * @objects SAI_OBJECT_TYPE_L2MC_GROUP
+     * @allownull true
+     * @default SAI_NULL_OBJECT_ID
+     */
+    SAI_VLAN_ATTR_UNKNOWN_UNICAST_FLOOD_GROUP,
```
Comment (lguohan) [inline inc/saivlan.h]: need another enum attribute to define where to flood to all vlan member, none or use l2mc group.

Comment (karthik-krishnamurthy) [inline inc/saivlan.h]: @lguohan  Added the enum. Please review

**Pull request merged**